atomicapp/constants.py

```diff
@@ -70,11 +70,11 @@
 }
 
 PROVIDERS = ["docker", "kubernetes", "openshift", "marathon"]
-PROVIDER_API_KEY = "providerapi"
-ACCESS_TOKEN_KEY = "accesstoken"
-PROVIDER_CONFIG_KEY = "providerconfig"
-PROVIDER_TLS_VERIFY_KEY = "providertlsverify"
-PROVIDER_CA_KEY = "providercafile"
+PROVIDER_API_KEY = "provider-api"
+ACCESS_TOKEN_KEY = "provider-auth"
```

Comment:
@dustymabe 
Yeah, I was thinking of doing that in a separate commit just in case I break anything. I agree though, changing it to PROVIDER_AUTH